segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/DurableLog.java

```diff
@@ -309,7 +303,8 @@ public int getId() {
 
     //region Recovery
 
-    private boolean performRecovery() throws Exception {
+    @SneakyThrows
```

Comment:
Looking at the code below, I think wrapping this in a RuntimeException might be more appropriate.